server/controllers/find/courseOfferingsController.ts

```diff
@@ -13,9 +14,17 @@ export default class CourseOfferingsController {
     return async (req: Request, res: Response) => {
       TypeUtils.assertHasUser(req)
 
-      const course = await this.courseService.getCourseByOffering(req.user.token, req.params.courseOfferingId)
-      const courseOffering = await this.courseService.getOffering(req.user.token, req.params.courseOfferingId)
-      const organisation = await this.organisationService.getOrganisation(req.user.token, courseOffering.organisationId)
+      const [course, [courseOffering, organisation]] = await Promise.all([
+        this.courseService.getCourseByOffering(req.user.token, req.params.courseOfferingId),
+        this.courseService.getOffering(req.user.token, req.params.courseOfferingId).then(async myCourseOffering => {
+          const myOrganisation = await this.organisationService.getOrganisation(
+            req.user.token,
+            myCourseOffering.organisationId,
+          )
+          return [myCourseOffering, myOrganisation] as [CourseOffering, Organisation]
+        }),
+      ])
```

Comment:
Re: https://github.com/ministryofjustice/hmpps-accredited-programmes-ui/pull/407#discussion_r1426719840

Maybe `transient`?
```suggestion
      const [course, [courseOffering, organisation]] = await Promise.all([
        this.courseService.getCourseByOffering(req.user.token, req.params.courseOfferingId),
        this.courseService.getOffering(req.user.token, req.params.courseOfferingId).then(async transientCourseOffering => {
          const transientOrganisation = await this.organisationService.getOrganisation(
            req.user.token,
            transientCourseOffering.organisationId,
          )
          return [transientCourseOffering, transientOrganisation] as [CourseOffering, Organisation]
        }),
      ])
```

Reply:
Let's leave it as it is. It's working around a restriction imposed by linting. Other projects use _ and __ and ___ depending on the levels, but the linting rule in this project also stops that.

Reply:
Sorry to jump in here, but I do think `_` would be better. This is a pattern we might have to re-use, and I'd rather we used an underscore rather than `my` in other places in the code - I think it's fine to disable that specific rule for this block with `/* eslint-disable no-underscore-dangle */` and `/* eslint-enable no-underscore-dangle */` at the end.